### ISSUE TITLE
"Step implementation missing for"

### ISSUE THREAD

**Issue by kantord** [TheBrainFamily/cypress-cucumber-preprocessor#360]
I have:

- Set up cypress-cucumber-preprocessor according to README
- Set `nonGlobalStepDefinitions` to `true`
- Put a step definitions into cypress/integration/common/i_open_course_test.js
- Created cypress/integration/coursePage.feature that uses that step

But I still get Step implementation missing for: I open /course/test :crying_cat_face: 

See my code: https://github.com/kantord/LibreLingo/pull/352/commits/90e82add9ad9e4a94dd49356eb7a05703b196227

Am I missing something?

**Comment by lgandecki** [comment]
Sorry for this. Could you go back to 2.1.0 and see if that fixes the problem?

**Comment by kantord** [comment]
I found that the problem has nothing to do with configuration issues.

For some reason,

```
Given("I open /course/test", () => {
    cy.visit("/course/test")
})
```

does not match

```
Given I open /course/test
```

however, if I change it to

```
Given("I open", () => {
    cy.visit("/course/test")
})
```

that matches 


```
Given I open
```

and everything works fine. I think the issue isn't in this repo therefore, and it was probably user error on my part.

However, maybe it would be nice to have some kind of feedback that I configured things correctly. I. e. I want to see a different error if no step definitions are found at all, vs. no matching step definition was found.
